I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml

```diff
@@ -431,7 +429,10 @@
  <section title="Optical Tributary Signal (OTSi)" anchor="OTSi">
   <t>
    The OTSi is defined in ITU-T Recommendation G.959.1, section 3.2.4
-   <xref target="G.959.1"/>. The YANG model defined below assumes that
+   <xref target="G.959.1"/> as "Optical signal that is placed within a 
+   network media channel for transport across the optical network." 
```

Comment:
```suggestion
   network media channel for transport across the optical network. This may consist of a single modulated optical carrier or a group of modulated optical carriers or subcarriers." 
```